docs/5.x/requirements.md

```diff
@@ -232,6 +232,24 @@ Gravity requires that these modules are loaded prior to installation.
 | Ubuntu-Core | 16.04 | br_netfilter, ebtable_filter, iptables, overlay |
 | Suse Linux (openSUSE and Enterprise) | 12 SP2, 12 SP3 | br_netfilter, ebtable_filter, iptables, overlay |
 
+### Inotify watches
+
+Kubelet configures multiple inotify watches per container so it's recommended
+to increase the `max_user_watches` kernel parameter to avoid running out of
+limits. The actual number is application-specific, but we recommended setting
+it to some large value:
```

Comment:
nit: I would hint at the existence of the inotify checker and the need to bump the value in that case.